core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -198,7 +198,8 @@ package object config {
 
   private[spark] val EVENT_LOG_ROLLING_MAX_FILE_SIZE =
     ConfigBuilder("spark.eventLog.rolling.maxFileSize")
-      .doc("The max size of event log file to be rolled over.")
+      .doc(s"Precondition: ${EVENT_LOG_ENABLE_ROLLING.key}. " +
```

Comment:
Okay, I saw the new PR https://github.com/apache/spark/pull/27575. We could match how we describe there. Shall we also fix `docs/configuration.md`?

Reply:
Nice catch! I missed we have equivalent description in doc as well. Will update as well.